_sass/single-page.scss

```diff
@@ -3,6 +3,16 @@
   gap: 3rem;
 }
 
+.single-page code {
+  padding: 1rem;
+  border-radius: 0.5rem;
+  font-size: 0.9rem;
+  background-color: #1a2421;
```

Comment:
@thisiskaransgit Do we have a theme to reference for this website?

Reply:
@Chadha93  PTAL